tests/libtest.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+# Test if config exists.
+config_exists(){
+  local config_dir="/var/lib/container-storage-setup"
```

Comment:
I might be wrong, but vivek once suggested to use local variables. Since their scope is well defined and limited. Global variables can be modified by any method, and it's harder to define their scope (who changed this global variable with what value kind of thing ?). Also if sourced, they can conflict with existing globals with same name. 

@rhvgoyal WDYT ? 

PS: @rhatdan It's a small change. I don't mind fixing it. Just need to know the standard we are following for future code changes.